### ISSUE TITLE
Login Interaction

### ISSUE THREAD

**Issue by alfonsusac** [alfonsusac/devscale-eventmaker#4]
- [x] Create a reusable function called `requestLogin()` that consists of the return of JSONified, awaited fetch() response to the login Endpoint.
- [x] The return has to be in the JSON datatype
- [x] The parameter has to accept 2 arguments: username and password
- [x] Put it in the `/lib/fetchAPI.js` file

Example:

```tsx
export async function requestLogin(){
  // ...
  return await res.json() 
}
```

- [x] Then call this function in the client component.
- [x] Make sure to show toast on succesfull login
- [x] Make sure to display toast on any error (using try catch)

**Comment by alfonsusac** [comment]
I pretend i didnt see you self-assigned this and im gonna assign @rizaardiyanto1412 to this assignee

**Comment by rizaardiyanto1412** [comment]
https://github.com/alfonsusac/devscale-eventmaker/pull/33

**Comment by rizaardiyanto1412** [comment]
@alfonsusac requestLogin() is causing an error from Eslint


![Image](https://github.com/alfonsusac/devscale-eventmaker/assets/83811723/47fe77ed-eae1-4002-905b-d22a56b9b001)


I change it to useRequestLogin

**Comment by alfonsusac** [comment]
I want to make the fetch into a requestLogin function, not the hooks itself, see coments in #33

**Comment by alfonsusac** [comment]
Closed in #33
